src/controllers/TransfersController.ts

```diff
@@ -20,7 +26,9 @@ export const TransfersController = {
         status,
       })
 
-      void reply.status(200).send(transfersResult)
+      const transfersWithValue = await coinMarketCapServiceInstance.appendConvertedAmountValueToTransfers(transfersResult)
```

Comment:
I think we have to do this while saving the transfer instead of in the controller.

Reply:
I agree here, we should save this value together with transfer as we want to show value at the time of bridging request happened.